mindsdb/integrations/handlers/email_handler/email_helpers.py

```diff
@@ -0,0 +1,83 @@
+import smtplib
+import imaplib
+import email
+from email.mime.multipart import MIMEMultipart
+from email.mime.text import MIMEText
+import pandas as pd
+
+
+class EmailClient:
+    def __init__(self, email, password, smtp_server='smtp.gmail.com', smtp_port=587, imap_server="imap.gmail.com"):
+        self.email = email
+        self.password = password
+        self.smtp_server = smtplib.SMTP(smtp_server, smtp_port)
+        self.imap_server = imaplib.IMAP4_SSL(imap_server)
+
+    def send_email(self, to_addr, subject, body):
+        msg = MIMEMultipart()
+        msg['From'] = self.email
+        msg['To'] = to_addr
```

Comment:
Is this working with multiple recipients?

Reply:
Yes, it works with a comma separated string. This is an example,
```
INSERT INTO email_datasource.emails(to, subject, body)
VALUES
("minurapunchihewa17@gmail.com, natashamp97@gmail.com", "TEST", "TEST")
```